### PR TITLE
Sprint 3/issue 30#home page

### PR DESCRIPTION
**Pull request by DevLuis123**
Home page completed!
image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (MauricioNC) [inline .vscode/settings.json:1]: What is this file for?

File: app/assets/stylesheets/show_reservation.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  margin-left: 0.5rem;
+  border-radius: 0.375rem;
+  padding-top: 0.75rem;
+  padding-bottom: 0.75rem;
+  padding-left: 1.25rem;
+  padding-right: 1.25rem;
+  background-color: #f3f4f6;
+  font-weight: 500;
+  display: inline-block;
+}
```
Comment (MauricioNC) [inline app/assets/stylesheets/show_reservation.css:64]: Add an empty line at the end of your code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/views/reservations/_search_filter_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        <%= form.label :active, 'Show only active reservations', class: 'ml-2 text-start font-semibold' %>
+      </div>
+    </div>
+
+    <div class='mt-4 mb-5 flex flex-col items-center min-[250px]:flex-row'>
+      <%= form.submit 'Search', class: 'rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium' %>
+      <%= link_to 'Clear', reservations_path, class: 'rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium ml-3 no-underline' %>
+    </div>
+  </div>
+<% end %>
```
Comment (MauricioNC) [inline app/views/reservations/_search_filter_form.html.erb:43]: Add an empty line at the end of your code

File: app/views/reservations/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      <h4><%= reservation.user.email %></h4>
+      <%= link_to "Show this reservation", reservation, class: "rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
+      <% if reservation.user.id == current_user.id || current_user.is_an_admin? %>
+        <%= link_to "Edit this reservation", edit_reservation_path(reservation), class: "rounded-lg py-3 ml-2 px-5 bg-gray-100 inline-block font-medium" %>
+      <% end %>
+      <hr class="mt-6">
+    <% end %>
+  </div>
+  <%== pagy_nav(@pagy) if @pagy.pages > 1 %>
+</div>
```
Comment (MauricioNC) [inline app/views/reservations/index.html.erb:29]: Add an empty line at the end of your code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Comment by MauricioNC** [comment]
This is a PR for a home page, nevertheless you are committing a lot of changes in different files. Remember that PRs must be small changes as well as their commits. 
It is too hard review all your code due to the quantity of changes, just keep this in mind for futures PRs.

Also you have many conflicts pending to resolve.

File: app/assets/stylesheets/home_page.css
Comment (PudinDeveloper) [inline app/assets/stylesheets/home_page.css:1]: Suggestion: Alphabetically order the CSS

File: app/assets/stylesheets/show_reservation.css
Comment (PudinDeveloper) [inline app/assets/stylesheets/show_reservation.css:1]: Suggestion: Alphabetically order the CSS

**Review by PudinDeveloper — APPROVED** [review]
Approved, only address the changes requested by others and I added some suggestions.

File: .vscode/settings.json (hunk 1 above)
Comment (DevLuis123) [reply .vscode/settings.json:1]: I don't know, but it has already been deleted!!

File: app/views/reservations/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,29 @@
+<div class="w-full">
+  <% if notice.present? %>
+    <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
+  <% end %>
+  <div class="flex justify-between items-center">
+    <h1 class="font-bold text-4xl">Reservations</h1>
+    <div>
+      <%= link_to "Home", root_path, class: "inline-block rounded-lg text-center py-3 px-5 bg-blue-600 text-white block font-medium" %>
+      <% if current_user.is_a_guest_or_admin? && Accommodation.all.count > 0 %>
+        <%= link_to "New reservation", new_reservation_path, class: "inline-block rounded-lg py-3 px-5 bg-blue-600 text-white block font-medium" %>
```
[the remark this answers is not in the document]
Comment (DevLuis123) [reply app/views/reservations/index.html.erb:10]: okay! we will solve it later

**Review by MauricioNC — APPROVED** [review]

**Review by AnaCarrizales — APPROVED** [review]

**Pull request merged**